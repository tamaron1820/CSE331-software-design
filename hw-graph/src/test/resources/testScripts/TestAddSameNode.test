# Tests that add Nodes correctly
# Test if same node is added graph should not change

# Create a graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n1

# Print the nodes in the graph and the outgoing edges from a and b
ListNodes graph1
ListChildren graph1 n1