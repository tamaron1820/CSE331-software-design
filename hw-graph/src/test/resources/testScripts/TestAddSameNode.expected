# Tests that add Nodes correctly
# Test if same node is added graph should not change

# Create a graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n1 to graph1

# Print the nodes in the graph and the outgoing edges from n1 and n2
graph contains: n1
the children of n1 in multiNodeGraph are: